packages/main/src/plugin/index.ts

```diff
@@ -509,7 +509,7 @@ export class PluginSystem {
       }
     });
 
-    statusBarRegistry.setEntry('help', false, 0, undefined, 'Help', 'fa fa-question-circle', true, 'help', undefined);
+    statusBarRegistry.setEntry('help', false, -1, undefined, 'Help', 'fa fa-question-circle', true, 'help', undefined);
```

Comment:
any reason why changing from 0 to -1 ?

Reply:
I mean, I see the comment 
> This PR also change the order priority of the help, to -1 to ensure it is always the one on the most right.

but should it be like that ?

Reply:
That's the easy fix I found to avoid this

<img width="212" alt="image" src="https://github.com/containers/podman-desktop/assets/42176370/182c1e21-1c40-43fc-a822-8a712e9adeff">

The task manager has some kind of triangle position, and with today implementation order, we register the task first or last, making it the most of the right

Reply:
😂 The task manager popup has no relationship to where its status bar item is, and the order of the other items is 'random'. Still trying to decide if this is brilliant or should be done 'correctly' to avoid future breakage.

Reply:
> Still trying to decide if this is brilliant or should be done 'correctly' to avoid future breakage.

I am fine with it :)